virtcontainers/pkg/oci/utils.go

```diff
@@ -594,6 +602,29 @@ func ContainerConfig(ocispec CompatOCISpec, bundlePath, cid, console string, det
 	return containerConfig, nil
 }
 
+func getShmSize(c vc.ContainerConfig) (uint64, error) {
+	var shmSize uint64
+
+	for _, m := range c.Mounts {
+		if m.Destination != "/dev/shm" {
+			continue
+		}
+
+		shmSize = vc.DefaultShmSize
```

Comment:
Why is this set inside the loop? Can't you just initialize `shmSize` to this value as currently if there is no host mount this function will return `0`?

Reply:
@jodh-intel I have added a `break` here. If no `shm` mount is found, then the `shmSize` will be zero, hence it is initialized to zero. In case host shm is to be used, this will be set to `DefaultShmSize`.